js/test/components/home.test.tsx

```diff
@@ -16,4 +42,19 @@ describe("home", () => {
     );
     expect(view.getByText(/fit for duty/i)).toBeInTheDocument();
   });
+
+  test("can reload the List on new sign-in", async () => {
+    const view = render(
+      <MemoryRouter>
+        <Home />
+      </MemoryRouter>,
+    );
+    await userEvent.click(
+      view.getByRole("button", { name: "Sign In Operator" }),
+    );
+
+    (fetch as jest.MockedFunction<typeof fetch>).mockClear();
```

Comment:
**question:** is it possible to use `jest.mocked(...)` here instead of having to use `as`?

Reply:
I KNEW there was something I forgot here. 56c3aef